tests/usage.test.ts

```diff
@@ -43,7 +43,7 @@ describe("parsing using the 'ajv' library", () => {
   examplesToTest.forEach(example => {
     test(`accepts a valid example: ${example.data.application.type.description}`, () => {
       // addFormats() is required for types UUID, email, datetime etc
-      const ajv = addFormats(new Ajv());
+      const ajv = addFormats(new Ajv({allowUnionTypes: true}));
```

Comment:
this fixes some warnings that were being thrown when testing ! see https://ajv.js.org/strict-mode.html#union-types